website/src/app/views/about/members.js

```diff
@@ -967,6 +968,12 @@ const members = {
       linkedinURL: 'https://www.linkedin.com/in/kyle-doliente-877133225/',
       imageURL: kyledoliente,
     },
+    {
+      name: 'Edward Wu',
```

Comment:
Sry just realized that your member info should be in the developer group. So move it into the developer group sorted by your first name. Then I can merge!

Reply:
Oh gotcha, I didn't realize that either